entities/Asigurare.java

```diff
@@ -0,0 +1,48 @@
+package entities;
```

Comment:
[Fix] toate pachetele trebuie sa vina sub cel principal, adica numele organizatiei pt care faci proiectul. In cazul nostru, o oraganizatie fictiva. Putem folosi com.company sau com.example, sau ro.unibuc. Vad ca tu ai deja com.company. pune inauntru si restul de pachete

Reply:
Multumesc frumos pentru feedback! O sa incerc sa repar. Nu prea stiam cum trebuie organizate fisierele, dar invatam :) .